app/services/idv/steps/threat_metrix_step_helper.rb

```diff
@@ -47,9 +47,19 @@ def threatmetrix_iframe_url(session_id)
         )
       end
 
-      def log_irs_tmx_fraud_check_event(result)
+      def log_irs_tmx_fraud_check_event(result, user)
         return unless IdentityConfig.store.irs_attempt_api_track_tmx_fraud_check_event
         return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
+
+        # This doesn't really feel like the right place, but there's not a more obvious one.
+        # We currently only need this for Attempts API stuff.
```

Comment:
^ What this comment says. This is the only place we need this, but it feels like this may be an unexpected side-effect. I think it also reinforces that "FraudEvent" is not a good name -- it's specific to a user undergoing a fraud check. I felt like I should move this up, but I never got to that, "Ohhh, _this_ is clearly where this belongs" point.

In fact, we only want this if `success` is false.